stacks/container-apps/cms.prx.org.yml

```diff
@@ -252,6 +252,11 @@ Resources:
               Value: !Ref SecretsVersion
             - Name: AWS_DEFAULT_REGION
               Value: !Ref AWS::Region
+            - Fn::If:
+                - CreateProductionResources
```

Comment:
In any case - aws-secrets is probably going to overwrite your ENV-of-the-same-name here.  But +1 safety!